docker-compose.yml

```diff
@@ -4,7 +4,9 @@ services:
     build: .
     ports:
       - "8000:8000"
-    command: python management_interface/manage.py runserver 0.0.0.0:8000
+    # probably a hack: createsuperuser would fail if issued on existing DB (as the user already exists)
+    # so to allow build command to finish and run the container, we force to ignore the error
+    command: sh -c "python management_interface/manage.py migrate && python management_interface/manage.py createsuperuser --noinput || true && python management_interface/manage.py runserver 0.0.0.0:8000"
```

Comment:
Let's go with it for now.  The way I've sorted out this sort of thing in the past is with a Makefile.  I'll knock one together and we can decide if we like it or not.